lib/metadata/util/md5deep.rb

```diff
@@ -107,6 +107,21 @@ def processDir(path, x, xmlNode)
       @fullDirCount += 1
     end
   end
+  
+  def processDirAsFile(path, x, xmlNode)
+    if x != "." && x != ".."
+      currDir = File.join(path, x)
+      begin
+        if isDir?(currDir)
+          xmlFileNode = xmlNode.add_element("file", "name" => x, "fqname" => currDir)
+          statHash = {}
+          statHash.merge!(getDirStats(currDir))
+          xmlFileNode.add_attributes(statHash)
+        end
+      rescue Errno::EACCES, RuntimeError, SystemCallError
```

Comment:
Codeclimate is complaining that you're ignoring these rescued errors.
Maybe you should just log the error at info level. That will quiet codeclimate and may also catch some unexpected behavior.